integrations/sumologic/sumologic-analytics/README.md

```diff
@@ -0,0 +1,87 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The Sumo Logic integration is a [Sensu Event Pipeline][pipeline] that enables real-time analytics and turn-key dashboards for Sensu Go.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `sumologic` ([pipeline])
+* `sumologic-metrics` ([sumologic-handler])
+* `sumologic-events` ([handler])
+* `sumologic_http_source_url` ([secret])
+* `sensu/sensu-sumologic-handler:0.3.0` ([asset])
+
+## Dashboards
+
+<!-- List of supported dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+This integration is compatible the the [Sumo Logic App Catalog][sumologic-app-catalog] — a library of over 250 turn-key dashboards for visualizing multi-cloud monitoring and observability data.
+
+**Example dashboard(s):**
+
+![](img/example-dashboard.png)
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Create a hosted collector & configure an HTTP Logs & Metrics source
+
+   [Create a Sumo Logic Hosted Collector][sumologic-hosted-collector] for Sensu Go, and configure an [HTTP Logs and Metrics Source][sumologic-logs-and-metrics-source].
+
+1. Add the Sumo Logic HTTP Logs & Metrics source URL to your secrets provider
+
+   Create an environment variable or Hashicorp Vault secret for the HTTP Logs & Metrics source URL.
+
+## Plugins
```

Comment:
So this one is weird... because it makes use of the commercial only  SumoLogicMetricsHandler available in the product for the metrics pipeline workflow.